mirgecom/initializers.py

```diff
@@ -0,0 +1,226 @@
+__copyright__ = (
+    "Copyright (C) 2020 University of Illinois Board of Trustees"
+)
+
+__license__ = """
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+"""
+
+import numpy as np
+import numpy.linalg as la  # noqa
+from pytools.obj_array import (
+    join_fields,
+    make_obj_array,
+    with_object_array_or_scalar,
+)
+import pyopencl.clmath as clmath
+from meshmode.mesh import BTAG_ALL, BTAG_NONE  # noqa
+
+# TODO: Remove grudge dependence?
+from grudge.eager import with_queue
+from grudge.symbolic.primitives import TracePair
+from mirgecom.eos import IdealSingleGas
+
+
+class Vortex2D:
```

Comment:
Docstring

Reply:
I'm responding here about documentation-related comments.   I've added a bunch of junk in docstrings comments to "quickly" address the no documentation issue; I have no math markup yet.  It would be nice to link the new domain/model document that will be created in Milestone1 to the documentation (and code) that implements each term - with eqns repeated in the docstring for the code construct.